src/algebra/field_power.lean

```diff
@@ -151,3 +144,35 @@ by rw [le_iff_lt_or_eq, le_iff_lt_or_eq, zpow_bit1_neg_iff, zpow_eq_zero_iff (bi
 lt_iff_lt_of_le_iff_le zpow_bit1_nonpos_iff
 
 end linear_ordered_field
+
+namespace tactic
+open positivity
+
+private lemma zpow_zero_pos [linear_ordered_semifield α] (a : α) : 0 < a ^ (0 : ℤ) :=
+zero_lt_one.trans_le (zpow_zero a).ge
+
+/-- Extension for the `positivity` tactic: raising a number `a` to an integer power `n` is positive
+if `n = 0` (since `a ^ 0 = 1`) or if `0 < a`, and is known to be nonnegative if
+`n = 2` (squares are nonnegative) or if `0 ≤ a`. -/
+@[positivity]
+meta def positivity_zpow : expr → tactic strictness
+| `(%%a ^ %%n) := do
+  n_typ ← infer_type n,
+  unify n_typ `(ℤ),
```

Comment:
Could you use `norm_num.derive` here to evaluate whether `n` is even/odd/zero, or is that too slow?

(Edit: I see the existing extension does the same for `ℕ` so I'd say don't invest too much time, apparently the simple way already works well enough.)

Reply:
As discussed in PM on Zulip, I think this is the role of a dedicated decision procedure. I am thus declaring this future work. Let's not reinvent Aesop quite yet :P